modules/services/udiskie.nix

```diff
@@ -73,7 +79,7 @@ in
         };
 
         Service = {
-          ExecStart = "${pkgs.udiskie}/bin/udiskie -2 ${commandArgs}";
+          ExecStart = "${pkgs.udiskie}/bin/udiskie -2 ${commandArgs} ${if cfg.sni then "--appindicator" else ""}";
```

Comment:
why put it there instead of the commandArgs. What is Sni ?

Reply:
I guess "status-notifier-item" from your toher pr :)

Reply:
commandArgs is generated with uniform mapping which adds unary minus to options. I'm not sure where it should be placed, but it seems like that you are right and it should be in commandArgs just by concatenating.

FYI. Sni is a standard for "modern tray" that comes after classical systray it used in KDE, gnome and taffybar f.e.